app/dashboard/templates/profiles/profile.html

```diff
@@ -234,10 +235,32 @@
                       </div>
                     </div>
                   </div>
-                </div>
-                {% endif %}
+                </div> {% endif %}
               </div>
             {% endfor %}
+		{% else %}
+		<BR>
```

Comment:
Can you change all occurrences into lowercase `<br>` ?

Reply:
Yes